libs/wing-fixture/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "wing-fixture",
+  "version": "0.0.0",
+  "description": "A fake library just for testing stuff out",
+  "author": {
+    "name": "Wing Cloud",
+    "email": "ping@monada.co",
+    "url": "https://wing.cloud",
+    "organization": true
+  },
+  "files": [
+    "**/*.js",
+    "**/*.w",
+    "!/target/**"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">=v18.0.0"
+  },
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/winglang/wing.git"
+  },
+  "scripts": {
+    "package": "npm pack"
+  },
+  "private": true,
+  "wing": {
+    "entrypoint": "lib.w"
+  },
```

Comment:
imo it might be nice to initially avoid any configuration for this, I think this default is reasonable.
The convention of `lib.w is the entrypoint of a library` should buy us time to better spec out the kind of options we want to be configurable.

Reply:
I also don't think we really need an entrypoint for libraries (chatted with @Chriscbr about this today), but I think we at least need a way to know that this is a wing library from reading `package.json`.

But it could be `wing: true`.